### PR TITLE
Adding eclypse scans

### PR DESCRIPTION
**Pull request by onelots**

### CODE REVIEW

File: src/web/mjs/connectors/Eclypse.mjs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+    constructor() {
+        super();
+        super.id = 'eclypse';
+        super.label = 'eclypse';
+        this.tags = [ 'manga', 'scanlation', 'webtoons', 'french' ];
+        this.url = 'https://eclypse-scan.com';
+    }
+}
+
```
Comment (Sheepux) [inline src/web/mjs/connectors/Eclypse.mjs]: Lint ;)

**Review by Sheepux — CHANGES_REQUESTED** [review]
Lint 
Connector img name should be lowercase

File: src/web/mjs/connectors/Eclypse.mjs (hunk 1 above)
Comment (onelots) [reply src/web/mjs/connectors/Eclypse.mjs]: I don't understand aha

Comment (Sheepux) [reply src/web/mjs/connectors/Eclypse.mjs]: check this ;)
https://github.com/manga-download/hakuneko/pull/4650/files

**Pull request closed without merging**